dev_tools/containers/ginkgo-cuda-base.py

```diff
@@ -17,12 +18,8 @@
 
 cuda_version = USERARG.get('cuda', '10.0')
 
-if float(cuda_version) < float(9.2):
-    image = 'nvidia/cuda:{}-devel-ubuntu16.04'.format(cuda_version)
-    Stage0.baseimage(image)
-else:
-    image = 'nvidia/cuda:{}-devel-ubuntu18.04'.format(cuda_version)
-    Stage0.baseimage(image)
+image = 'nvidia/cuda:{}-devel-ubuntu16.04'.format(cuda_version)
```

Comment:
So, we use 16.04 for all containers now ?

Reply:
Yes, this is required for Intel Compilers 2017 and 2018. See the text of commit bbe81a9.